hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1533,7 +1533,7 @@
     {
       name: chip_sw_kmac_app_rom
       uvm_test_seq: chip_sw_base_vseq
-      sw_images: ["//sw/device/tests:kmac_app_rom_test:1:new_rules"]
+      sw_images: ["//sw/device/tests:kmac_app_rom_test_sim_dv:1:new_rules"]
```

Comment:
are you sure you need the suffix `_sim_dv`? I believe this gets automatically here: https://cs.opensource.google/opentitan/opentitan/+/master:hw/dv/tools/dvsim/sim.mk;drc=30d7e787c753caaa03fe68a4a70da1bbcbc1d96f;l=87

Reply:
It is necessary in this case because we have to manually create one `opentitan_test` per exec-env. I'll add a comment for avoid any confusion.

Reply:
hmm ok, just to confirm this works b/c the sim.mk file I referenced is super brittle, did you try to run this in DV with the current label?

Reply:
Yes I ran it already to check